### DISCUSSION TITLE
Help Wanted: Changes in urql 4.0.7 break gqlgen integration tests

### DISCUSSION THREAD

**Discussion by StevenACoffman** [99designs/gqlgen#2997] [Announcements]
Hey folks,
I'm on vacation, so if anyone can please take a look at the PR that bumps urql from 4.0.6 to 4.0.7 in /integration causing failures and make a PR that corrects our usage, I would really appreciate it!

See https://github.com/99designs/gqlgen/pull/2995

**Comment by erka** [comment]
Probably #2998 should be enough to resolve it

**Comment by StevenACoffman** [comment]
@erka (and @UnAfraid too!) Thanks so much for looking at this while I was on my road trip!
